Doc/library/pathlib.rst

```diff
@@ -971,13 +971,13 @@ call fails (for example because the path doesn't exist).
    list of subdirectories is retrieved before the tuples for the directory and
    its subdirectories are walked.
 
-   When *top_down* is True, the caller can modify the *dirnames* list in-place
+   When *top_down* is ``True``, the caller can modify the *dirnames* list in-place
```

Comment:
In general, we like docs to say `something is true` so that it flows better (code bit stops normal reading) and also implies that the value is duck typed (true-ish, any object that evaluates to True in a boolean context), as is our common practice.  In other words, it is rarely needed to be extremely exact about True/False values.

Reply:
Apologies for leading you in the other direction earlier, @Ovsyanka83. I was aiming for consistency but didn't know about this ^

Reply:
My problem is that it won't really flow better in cases where we say "set ... to true"

Reply:
You can have both; you can talk about the default value being ``True``, but when you're talking about what the value does, it's preferred to talk about "true" and "false" things instead of ``True`` and ``False``.